amino.go

```diff
@@ -87,6 +88,9 @@ const (
 	//Typ3_Interface  = Typ3(7)
 )
 
+// we use this time to init. a zero value (opposed to reflect.Zero which gives time.Time{} / 01-01-01 00:00:00)
+var zeroTime, _ = time.Parse("2006-01-02 15:04:05 +0000 UTC", "1970-01-01 00:00:00 +0000 UTC")
```

Comment:
It's not very likely but ignoring the error is not advisable. There are multiple ways to do this, one is to use a global and then have an init function which panics if the parse fails, or have a little wrapper like `mustParse` which panics in case of an error. The latter is desirable as use of `init` needs to be considered and initialisation order needs to be kept in mind.

Those strings would make for great constants.

Reply:
Thanks! Very good suggestions 👍